src/rsz/src/Resizer.cc

```diff
@@ -263,7 +255,69 @@ void Resizer::removeBuffers()
   logger_->info(RSZ, 26, "Removed {} buffers.", remove_count);
 }
 
-bool Resizer::bufferBetweenPorts(Instance* buffer)
+LibertyCell* Resizer::findSmallestEquiv(const DcalcAnalysisPt* dcalc_ap,
```

Comment:
Strictly speaking this is the weakest not the smallest though those are generally correlated.  If you truly want the smallest you can take the area from LEF (odb) and it will work on multi-output gates as well.  Liberty may also have area but it is optional